pkg/kubelet/image_manager_test.go

```diff
@@ -37,6 +37,7 @@ func newRealImageManager(policy ImageGCPolicy) (*realImageManager, *container.Fa
 	return &realImageManager{
 		runtime:      fakeRuntime,
 		policy:       policy,
+		minAge:       0,
```

Comment:
One way to do it: https://github.com/ssoroka/ttime
See also https://groups.google.com/forum/m/#!topic/golang-nuts/Mkonfn4j4jo